components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -86,6 +86,8 @@ $env:HAB_BLDR_URL = "$script:HAB_BLDR_URL"
 if (!(Test-Path Env:\HAB_BLDR_CHANNEL)) {
     $env:HAB_BLDR_CHANNEL = "stable"
 }
+# Enable the `--ignore-local` option for `hab pkg install`
+$env:HAB_FEAT_IGNORE_LOCAL = "true"
```

Comment:
this would "work" but the powershell convention would be `$env:HAB_FEAT_IGNORE_LOCAL = $true`